go.mod

```diff
@@ -51,6 +51,7 @@ require (
 	github.com/containerd/log v0.1.0 // indirect
 	github.com/dgryski/go-rendezvous v0.0.0-20200823014737-9f7001d12a5f // indirect
 	github.com/distribution/reference v0.6.0 // indirect
+	github.com/evanphx/json-patch v5.9.0+incompatible // indirect
```

Comment:
Is this really needed? It is an indirect dependency so no under our direct control, but it looks weird